TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Induction.cshtml.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail;
+
+public class InductionModel(TrsDbContext dbContext) : PageModel
+{
+    [FromRoute]
+    public Guid PersonId { get; set; }
+
+    public InductionStatus Status { get; set; }
+
+    public DateOnly? StartDate { get; set; }
+
+    public DateOnly? CompletionDate { get; set; }
+
+    public IEnumerable<InductionExemptionReasons>? ExemptionReasons { get; set; }
```

Comment:
`InductionExemptionReasons` is a `Flags` enum so only need a single value here instead of a collection.